### PR TITLE
TST: clean up pysat install on travis

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Pysat 2.1.0 is now available via pip.  There is no longer a need for the custom pysat install, so the pysat family of codes is being upgraded to a "standard" installation through pip.  This will allow the `no_sgp4` branch of pysat to be deleted.

## Type of change

- Test environment update

# How Has This Been Tested?

Updates to Travis-CI environment, so tested there.

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``master``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

**Comment by jklenzing** [comment]
Recurring error in python 3.x tests on travis:
```
ModuleNotFoundError: No module named 'pysat.instruments.pysat_testing'
```

**Comment by jklenzing** [comment]
Seems to be an issue with the local install of pysat.  Tested on my computer (python 3.7.3, pandas 0.24.2) and everything works fine with the pip-installed version of pysat 2.1.0.

**Comment by jklenzing** [comment]
> Seems to be an issue with the local install of pysat. Tested on my computer (python 3.7.3, pandas 0.24.2) and everything works fine with the pip-installed version of pysat 2.1.0.

Problem solved: the pip installed version does not include the testing instruments.

**Comment by jklenzing** [comment]
Rewriting to use master branch from git.

**Comment by jklenzing** [comment]
Two new updates based on Travis CI config validation suggestions:
- Specify os as linux
- `matrix` has been replaced by `jobs`

File: .travis.yml [hunk 1]
```diff
@@ -1,42 +1,39 @@
...
     - python: 3.7
+    - python: 3.8
 
 services: xvfb
 addons:
   apt:
     packages:
     - gfortran
-
+
```
Comment (aburrell) [inline .travis.yml]: Remove tab

File: .travis.yml [hunk 2]
```diff
@@ -1,42 +1,39 @@
...
-  - echo 'installing pysat'
   - cd ./pysat
-  - git checkout develop
   # set up data directory
   - mkdir /home/travis/build/pysatData
   # install pysat
   - python setup.py install >/dev/null
   - cd ../pysatSeasons
-
+
```
Comment (aburrell) [inline .travis.yml]: Remove space

**Comment by jklenzing** [comment]
Closing this in lieu of changes in #16

**Pull request closed without merging**